### PR TITLE
Dp 2117 topic page fixes

### PR DESCRIPTION
**Pull request by jesconstantine**
WIP PR to show updates to Topic page based on feedback for #380 

Conversation and review happening here: https://github.com/massgov/mayflower/pull/380/files

### CODE REVIEW

File: styleguide/source/assets/scss/02-molecules/_section-links.scss [hunk 1]
```diff
@@ -63,7 +63,7 @@
     }
 
     @media ($bp-x-small-min) {
-      margin: 1.4em auto -40px;
+      margin: -40px auto 1.4em;
```
Comment (jesconstantine) [inline styleguide/source/assets/scss/02-molecules/_section-links.scss:66]: This fixes the Section links with icons (like on section landing pages) layout which was broken once I implemented flex box to add the more link.  

Screenshot of issue:
image

Fixed:
image

**Comment by gender404** [comment]
Is this ready for review?

**Comment by jesconstantine** [comment]
@legostud sure, I was responding to the comments over in the merged PR, if you want to see how commits address your comments/questions let me know.

**Comment by gender404** [comment]
I noticed this is a merge to Master, please merge this into dev once prod has been updated with a new tag.

**Pull request merged**